transifex/utils.py

```diff
@@ -301,21 +304,14 @@ def merge_pr(self):
         while retries <= self.MAX_MERGE_RETRIES:
             # Assumes only one commit is present on the PR.
             time.sleep(60 * 5)
-            state = self._get_tests_combined_status()
 
-            if state == 'failure':
-                logger.error(
-                    'A failing CI build prevents [%s/#%d] from being merged. Notifying %s.',
-                    self.name, self.pr.number, self.owner
-                )
+            # Update our local PR object from the server (undocumented in PyGithub?)
+            self.pr.update()
+            # Not fully documented in regular API, but this looks like the same thing in the GraphQL docs:
+            # https://docs.github.com/en/graphql/reference/enums#mergestatestatus
+            state = self.pr.mergeable_state
```

Comment:
Consider opening issues around these questions or lacks, and pointing to the opened issues.

Reply:
I think it's OK to leave it as "it will work until it breaks" since this is just a way to avoid trying to merge too early; GitHub's own mergeability checks are the final backstop.

I've filed an issue on PyGithub for documenting the method, and I'll add links to that and to the Github discussion around their docs on `mergeable_state`, but I don't think we need to track it in our own issues.

Reply:
(In retrospect, I guess you weren't suggesting filing Jira tickets, heh.)

Reply:
Correct. Just github issues where and if appropriate.